charts/prometheus-node-exporter/Chart.yaml

```
@@ -6,7 +6,7 @@ keywords:
   - prometheus
   - exporter
 type: application
-version: 4.26.0
+version: 4.27.0
```

A fix should bump to patch version.

```suggestion
version: 4.26.1
```